README.md

```diff
@@ -99,6 +99,64 @@ gulp watch
 
 While it runs, it watches all files for changes and automatically updates your browser.
 
+### Installing Concepts OWA on OpenMRS Reference Application
+
+First, deploy concept dictionary to server, It can be done manually or automatically:
+
+<b>Manual deploy:</b><br>
+While in openmrs-owa-conceptdictionary directory, make ditributable *.zip package by running this gulp command:
+````sh
+gulp build
+````
+or this command, for minified production build
+````sh
+gulp build-prod
+
+````
+It will create `/dist` folder in project directory, which contains `conceptdictionary.zip` package.
+To add this package to Reference App, enter advanced administration panel (default at: `openmrs/admin/index.htm`), then go to `Manage Apps` in `Open Web Apps Module` menu, click "Browse", find `conceptdictionary.zip` file and then upload it via button.
+
+<b>Automatic deploy:</b><br>
+<p>Note: First please make sure the 'APP_ENTRY_POINT' in 'config.json' is set, see <b>Local Deploy</b> for more info</p>
+While in openmrs-owa-conceptdictionary directory, run following gulp command:
+````sh
+gulp deploy-local
+````
+It will automatically create distributable package and upload it to local server.<br>
```

Comment:
The above steps are for developers. End users do not need to know anything about building the app. They start from downloading a zip (we don't have a link yet so put a placeholder) and uploading it via the UI.